app/models/event.rb

```diff
@@ -28,6 +28,8 @@ class Event < ActiveRecord::Base
   has_many :participant_groups
   has_many :date_ranges
   accepts_nested_attributes_for :date_ranges
+  validates_presence_of :name
+  validates_presence_of :description
```

Comment:
This might be nitpicking, but in other models we put all "validates_presence_of" in one line, separated by commas.